### PR TITLE
feat: Transaction history table

### PR DESCRIPTION
**Pull request by saraileon**
- Add transaction history table
- Refactor status badge to show different colors and enable/disable pulse based on props
- Remove address label to show it in full

transaction_history_table2

Screenshot_2417

### CODE REVIEW

**Review by luisgj — APPROVED** [review]

File: src/components/TransactionsTable/TableComponets.tsx [hunk 1]
```diff
@@ -0,0 +1,168 @@
+import Box from '@mui/material/Box';
```
Comment (luisgj) [inline src/components/TransactionsTable/TableComponets.tsx:1]: typo in the file name

File: src/components/TransactionsTable/TransactionRow.tsx [hunk 2]
```diff
@@ -0,0 +1,232 @@
...
+              display: 'flex',
+              flexDirection: 'row',
+              gap: 3,
+            }}
+          >
+            <>
+              <RowTextDetail>
+                <span>{t('table.fee')}</span>
+                {row.fee ? formatNumber(row.fee) : t('table.noFee')}
+              </RowTextDetail>
```
Comment (luisgj) [inline src/components/TransactionsTable/TransactionRow.tsx:201]: the fee needs to be hidden. hehe sorryy 😬

File: src/pages/Transactions/History/index.tsx [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+        <Typography
+          variant="body1"
+          sx={{
+            fontSize: '27px !important',
+            fontFamily: 'Kanit',
+            fontWeight: '500',
+            mb: 1,
+          }}
+        >
+          Transaction History
```
Comment (luisgj) [inline src/pages/Transactions/History/index.tsx:29]: lets add a translation for this

File: src/components/TransactionsTable/TableComponets.tsx (hunk 1 above)
Comment (saraileon) [reply src/components/TransactionsTable/TableComponets.tsx]: Done

**Review by luisgj — APPROVED** [review]

**Pull request merged**